### DISCUSSION TITLE
Loading of modules / functions from .NET ?

### DISCUSSION THREAD

**Discussion by BambOoxX** [HyperSphereStudio/JULIA.Net#5] [Q&A] [answered]
Hello, 

Congrats for all your work, it already looks promising ! 

I have some questions about the current capabilities, as I did not find examples of this in you tests (but maybe I looked the wrong way ^^)
- Is it possible to tell julia to load a module (maybe it should be done with `Julia.Eval("using Module")`)
- How do you use a function from a module (`Julia.Eval("Module.function()")` ?)

**Comment by HyperSphereStudio** [comment, marked as answer]
At the moment module loading into the main module can only be done as you said using Julia.Eval("using Module"). Thats a very small expression so I do not see much a performance benefit from doing it via c. I can add a function into JLModule to do this for your if you want though.

Refer to JLFun.cs for your answer to #2.
```csharp
  var StringF = JLModule.Base.GetFunction("string");
   //Then used like StringF.Invoke(my_obj);
```
